src/components/imports/Imports.js

```diff
@@ -0,0 +1,4 @@
+export { default as NavBar } from '../re-usable/Navbar';
```

Comment:
It is best to use the camel case for file naming convention or at least stick with one pattern for consistency

Reply:
Noted